misk-api/src/main/kotlin/misk/scope/ActionScoped.kt

```diff
@@ -4,6 +4,8 @@ package misk.scope
 interface ActionScoped<out T> {
   fun get(): T
 
+  fun getIfInScope(): T? = get()
```

Comment:
Added a default return value, so that it's not a breaking change for any other classes implementing this interface across cash app repos.